guvnor-asset-mgmt/guvnor-asset-mgmt-client/src/test/java/org/guvnor/asset/management/client/editors/project/structure/ReleaseRepositoryPresenterTest.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2014 JBoss Inc
```

Comment:
I am sure this is easily configurable in all major IDEs. You can define template and it will put the current year there when creating the class.